service/create/service.go

```diff
@@ -243,7 +239,7 @@ func (s *Service) Boot() {
 						policy = &awsresources.Policy{
 							ClusterID: cluster.Spec.Cluster.Cluster.ID,
 							KMSKeyArn: kmsKey.Arn(),
-							S3Bucket:  s.s3Bucket,
+							S3Bucket:  cluster.Name,
```

Comment:
After a moment of thought I unfortunately need to disagree with that idea.

When we will create buckets per customer, not per cluster, what would delete those buckets? Delete operator couldn't really do that, because it's supposed to delete only the stuff directly related to the cluster. We could check whether the cluster we are deleting is the last one from the customer, but it sounds like an anti-pattern to me. I think we shouldn't make cluster deletion that complicated.

That said, I would like to stick just with the cluster name.